SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1046,6 +1063,13 @@ void USpatialNetDriver::NotifyActorDestroyed(AActor* ThisActor, bool IsSeamlessT
 
 void USpatialNetDriver::Shutdown()
```

Comment:
Do we have any ordering guarantees in that we will always get `NetDriver1::Init -> NetDriver1::Shutdown -> NetDriver2::Init -> NetDriver2::Shutdown` and not, say, `NetDriver1::Init -> NetDriver2::Init -> NetDriver1::Shutdown`?

Reply:
AFAIK, each game instance owns its own net driver. So ordering should not have consequences.